ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/metadata/MetadataPresenterImpl.java

```diff
@@ -89,26 +155,35 @@ public void onFailure(Throwable caught) {
             }
         };
 
-        drService.setDiskResourceMetaData(resource,
-                                          view.getMetadataTemplate(),
-                                          view.getAvus(),
-                                          batchAvuCallback);
+        DiskResourceUserMetadata umd =
+                AutoBeanCodex.decode(autoBeanFactory, DiskResourceUserMetadata.class, "{}").as();
+        umd.setAvus(view.getUserMetadata());
+        drService.setDiskResourceMetaData(resource, umd, view.getAvus(), batchAvuCallback);
```

Comment:
Yup. I discussed this with @ash3rz when reveiwing her belphegor changes but decided we will address after 2.7. I will look into to see if I can get it done before last drop and into 2.7